ansible/files/whisks_design_document_for_activations_db.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
When you use wipedb, there is no check if `split` is false or true.
The reason is, that I want to remove this flag in one of my next commits to maintain only the version with the splitted database.